RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -200,6 +200,20 @@ - (bool) isTablet
   return [[UIDevice currentDevice] userInterfaceIdiom] == UIUserInterfaceIdiomPad;
 }
 
+- (CGFloat) width
```

Comment:
This name looks a bit too generic to me.

Additionally, would you consider merging `height` and `width` into `getInitialScreenSize` on the native side as well? I think returning a `NSDictionary` should be the way to go